google-cloud-gaming-v1/synth.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This script is used to synthesize generated parts of this library."""
+
+import synthtool as s
+import synthtool.gcp as gcp
+import synthtool.languages.ruby as ruby
+import logging
+
+
+logging.basicConfig(level=logging.DEBUG)
+
+gapic = gcp.GAPICMicrogenerator()
+library = gapic.ruby_library(
+    "gaming", "v1",
+    extra_proto_files=["google/cloud/common_resources.proto"],
+    generator_args={
+        "ruby-cloud-gem-name": "google-cloud-gaming-v1",
+        "ruby-cloud-title": "Cloud Gaming V1",
+        "ruby-cloud-description": "Deploys and manages infrastructure for global multiplayer gaming experiences.",
```

Comment:
"Game Servers fully manages Agones, an open source game server management project that runs on Kubernetes."

from https://cloud.google.com/game-servers

Reply:
Yeah I saw that description too, but I found it awkward, possibly because you usually hear "fully managed" used as an adjective rather than "fully manage" used as a verb. (Indeed, it would probably be better rendered "manages Agones fully.")

If we want to mention Agones, how about "With Game Servers, studios and publishers can deploy and manage their game server infrastructure hosted on multiple Agones clusters around the world through a single interface." which comes from https://cloud.google.com/game-servers/docs

Reply:
:+1 I like that one. Mainly I think it's important that it contains the strings "Game Servers" and "Agones" for search discovery.